src/sentry/static/sentry/app/views/releasesV2/list/clippedHealthRows.tsx

```diff
@@ -43,16 +41,27 @@ class clippedHealthRows extends React.Component<Props, State> {
     const {children, maxVisibleItems, fadeHeight, className} = this.props;
     const {collapsed} = this.state;
 
+    const displayCollapsedButton = !collapsed && children.length > maxVisibleItems;
+
     return (
-      <Wrapper className={className}>
+      <Wrapper
+        className={className}
+        fadeHeight={fadeHeight}
+        displayCollapsedButton={displayCollapsedButton}
+        height={
```

Comment:
this will only show the first 4 items on mobile devices even if the maxVisibleItems is over 4